.github/workflows/quick.yaml

```diff
@@ -127,11 +127,18 @@ jobs:
           sudo sed --in-place -E 's/# (deb-src.*updates main)/  \1/g' /etc/apt/sources.list
           sudo apt-get --quiet=2 update
           sudo apt-get --quiet=2 build-dep squid
-          sudo apt-get --quiet=2 install linuxdoc-tools libtool-bin ${{ matrix.compiler.CC }}
+          sudo apt-get --quiet=2 install linuxdoc-tools libtool-bin ${{ matrix.compiler.CC }} ccache
 
       - name: Checkout sources
         uses: actions/checkout@v4
 
+      - name: Setup ccache
+        uses: squid-cache/ccache-action@v1.2.14
+        with:
+          verbose: 2 # default 0
+          append-timestamp: false
+          key: ${{ github.job }}-${{ matrix.os }}-${{ matrix.compiler.CC }}-${{ matrix.layer.nick }}
+
       - name: Run build on Linux
         run: ./test-builds.sh ${{ matrix.layer.name }}
```

Comment:
Why are we not adding ccache to all existing build tests, but not MacOS tests? Ccache Action [docs](https://github.com/squid-cache/ccache-action) mention that the Action "works" on MacOS.

Reply:
My own experience with ccache is mixed, it hasn't worked well for me so I am not using it on that OS